File: Modul 2/Soal 5/PRAK205-2310817210019-NaufalElyzar.c
#include <stdio.h>
#include <math.h>

int main() {
    double A, B, C, L, K;

    scanf("%lf", &A);
    scanf("%lf", &B);

    C = sqrt((B * B) - (A * A));
    K = A + B + C;
    L = 0.5 * A * C;

    printf("Alas     : %.0lf\n", C);
    printf("Tinggi   : %.0lf\n", A);
    printf("Keliling : %.0lf\n", K);
    printf("Luas     : %.0lf cm^2\n", L);
    
    return 0;
}

File: Modul 2/Soal 5/PRAK205-2310817210019-NaufalElyzar.py
import math

A = float(input())
B = float(input())

C = math.sqrt((B ** 2) - (A ** 2))
K = A + B + C
L = 0.5 * A * C

print(f"Alas     : {C:.0f}")
print(f"Tinggi   : {A:.0f}")
print(f"Keliling : {K:.0f}")
print(f"Luas     : {L:.0f} cm^2")